### PR TITLE
YC-861: Do not allow to send an custom email when status changed to received

### PR DESCRIPTION
**Pull request by rbovard**
Fix https://jira.liip.ch/browse/YC-861

Please review.

### CODE REVIEW

File: geocity/apps/core/static/js/permit_request_amending.js [hunk 1]
```diff
@@ -16,7 +25,8 @@ notify.addEventListener('change', (e) => {
 
 changeReasonDisplay()
 
-if (parseInt(statusChange.value) === COMPLEMENTARY_INFO) {
+if (parseInt(statusChange.value) === STATUS_AWAITING_SUPPLEMENT ||
+  parseInt(statusChange.value) === STATUS_RECEIVED) {
   statusChange.dispatchEvent(new Event("change"))
 }
```
Comment (monodo) [inline geocity/apps/core/static/js/permit_request_amending.js:32]: This does not work as expected: checkbox is always check when visible

image

**Review by monodo — CHANGES_REQUESTED** [review]

File: geocity/apps/core/static/js/permit_request_amending.js (hunk 1 above)
Comment (rbovard) [reply geocity/apps/core/static/js/permit_request_amending.js:32]: It wasn't the case before my change?

Comment (rbovard) [reply geocity/apps/core/static/js/permit_request_amending.js:37]: I review the logical behind it

**Comment by rbovard** [comment]
@monodo Thanks, review adressed. Please review again.

**Review by monodo — APPROVED** [review]

**Pull request merged**